docs/how_to_guides/msm_estimation_exercise.ipynb

```diff
@@ -0,0 +1,1419 @@
+{
```

Comment:
<p>For bigger numbers, <code>10_000</code> is easier to read than <code>10000 </code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/respy/pull/356/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>

Reply:
Didn't know this was possible, that is definitely more readable. Thanks!